.tekton/tasks/pipeline-service-tests.yaml

```diff
@@ -6,10 +6,25 @@ metadata:
 spec:
   params:
     - name: target_branch
+    - name: branch
   workspaces:
     - name: kubeconfig-dir
     - name: source
   steps:
+    - name: checkout-plnsvc-code
```

Comment:
This step is not needed, because the cloned codes already is copy to docker container at this [step](https://github.com/openshift-pipelines/pipeline-service/blob/main/.tekton/tasks/ci-runner-container-setup.yaml#L36)

Reply:
This is the bit that will make sure that the content on the ci-runner matches the version we want to test.

Reply:
ok, I updated my PR https://github.com/openshift-pipelines/pipeline-service/pull/758, i think it should solve the issue.